modules/dnn/perf/perf_net.cpp

```diff
@@ -202,6 +202,18 @@ PERF_TEST_P_(DNNTestNetwork, YOLOv3)
     processNet("dnn/yolov3.weights", "dnn/yolov3.cfg", "", inp / 255);
 }
 
+PERF_TEST_P_(DNNTestNetwork, YOLOv4)
+{
+    if (backend == DNN_BACKEND_HALIDE)
+        throw SkipTestException("");
+    if (target == DNN_TARGET_MYRIAD)
+        throw SkipTestException("");
+    Mat sample = imread(findDataFile("dnn/dog416.png"));
```

Comment:
Just to match origin behavior, we can also apply
```cpp
cvtColor(sample, sample, COLOR_BGR2RGB)
```
becase YOLOs work with RGB image.

Reply:
Fixed YOLOv3 too